### PR TITLE
Add IO key to it.piattaforme

### PR DESCRIPTION
**Pull request by mspasiano**

### CODE REVIEW

File: docs/it/schema.it.rst
Comment (bfabio) [inline docs/it/schema.it.rst:122]:
```suggestion
Se presente e impostato a ``true``, il software si interfaccia con `IO - l'app dei servizi pubblici <https://io.italia.it/>`__.
```

**Review by bfabio — CHANGES_REQUESTED** [review]
So it is coherent with #141

**Comment by bfabio** [comment]
Thanks for the PR @mspasiano, can you please rebase it on main so that we can get the preview?

**Comment by bfabio** [comment]
@yaml-9000 national-section ruphy
@yaml-9000 minor-change

**Comment by yaml-9000** [comment]
This is now marked as a national section change to the Standard and it will require the approval of the Steering Committee member representing the Country (@ruphy) and at least 50% approve votes.

[National section rules](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/procedure-proposing-changes-and-voting.md#country-specific-sections)

cc @ruphy @ruphy @publiccodeyml/steering-committee


<sub>[:page_facing_up: Voting procedure](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/procedure-proposing-changes-and-voting.md#voting) | [:page_facing_up: Working Group Charter](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/charter.md) | [:robot: bot commands](https://github.com/publiccodeyml/bot#commands)</sub>

**Comment by yaml-9000** [comment]
Thanks for your contribution :pray:

This is now marked as a [`minor-change` proposal to the standard](https://github.com/publiccodeyml/publiccode.yml/labels/standard-minor-change),
this means that old versions of `publiccode.yml` will still be valid with this change.

Example of minor changes are additions of new keys or making keys optional.

The Chair will eventually pick up this proposal and start the voting procedure using `@yaml-9000 vote-start`

cc @ruphy @publiccodeyml/steering-committee


<sub>[:page_facing_up: Voting procedure](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/procedure-proposing-changes-and-voting.md#voting) | [:page_facing_up: Working Group Charter](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/charter.md) | [:robot: bot commands](https://github.com/publiccodeyml/bot#commands)</sub>

**Comment by ruphy** [comment]
@yaml-9000 vote-start

**Comment by yaml-9000** [comment]
**Voting is now open on this proposal!**

If you are a member of the [Steering Committee](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/charter.md#steering-committee-publiccodeymlsteering-committee) you can now vote!

The polls will stay open for 14 days, until **Wed, 27 Apr 2022 13:03:42 GMT**.
At the end of that period the Chair (@ruphy) will mark the voting period as over using `@yaml-9000 vote-end`

Leave a :+1: (thumbs up) **on this comment** to accept the proposal or a :-1: (thumbs down) to reject it.

cc @publiccodeyml/steering-committee


<sub>[:page_facing_up: Voting procedure](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/procedure-proposing-changes-and-voting.md#voting) | [:page_facing_up: Working Group Charter](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/charter.md) | [:robot: bot commands](https://github.com/publiccodeyml/bot#commands)</sub>

**Comment by yaml-9000** [comment]
# Vote results

The results of [the vote](https:&#x2F;&#x2F;github.com&#x2F;publiccodeyml&#x2F;publiccode.yml&#x2F;pull&#x2F;139#issuecomment-1098024598) are:

|Vote|Members|Votes count|Percentage|
|-|-|-|-|
|:+1: Approve | @ruphy @bzg   | 2   | 100.0%   |
|:-1: Reject  |  | 0 | - |



**Proposal approved** :+1:

This proposal is now ready to be merged and get released with a new version of the standard.
      

cc @ruphy @publiccodeyml/maintainers


<details>
  <summary><sub>Details<sub></summary>
  <br>
  <p>
    <strong>National section vote</strong>: approve vote by ruphy required and at least <strong>50%</strong> approve votes
  </p>
  <p>
    The following users voted (includes non-members of the steering committee):
  </p>

  - <strong>mspasiano</strong> voted :+1:

- <strong>ruphy</strong> voted :+1:

- <strong>bzg</strong> voted :+1:
</details>


<sub>[:page_facing_up: Voting procedure](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/procedure-proposing-changes-and-voting.md#voting) | [:page_facing_up: Working Group Charter](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/charter.md) | [:robot: bot commands](https://github.com/publiccodeyml/bot#commands)</sub>

**Review by bfabio — APPROVED** [review]

**Pull request merged**
